.env

```diff
@@ -13,13 +13,14 @@ RUNTIME_MIN_REQUESTS_PER_NODE="200" # Required, The min threshold for the number
 RUNTIME_MAX_REQUESTS_PER_NODE="400" # Required, The max threshold for the number of requests per runtime node. Exceeding this for 2 times in 5 minutes will scale up the number of runtime nodes by 1
 
 # Common
-AWS_VPC_ID=""                            # VpcId of your existing Virtual Private Cloud (VPC) in the AWS account. If left empty, then a new VPC will be created automatically.
-AWS_VPC_DEPLOYMENT_SUBNETS_TYPE="PUBLIC" # Required, PRIVATE or PUBLIC
-AWS_EC2_KEY_PAIR_NAME=""                 # Required, The EC2 Key Pair to allow SSH access to the ec2 instances. More info : https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/create-key-pairs.html.
-AWS_CERTIFICATE_ARN=""                   # optional, The ARN of the certificate to be used by the application load balancer, More info : https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html.
-AWS_EFS_DNS=""                           # optional, The EFS DNS for the file system containing plugins and template/translation overrides.
-LOADBALANCER_IP_RANGE_CIDR=""            # optional, The IP address range that can be used to access Curity Runtime service through the load balancer, in CIDR format. Defaults to 0.0.0.0/0
-TRUSTED_IP_RANGE_CIDR=""                 # optional, for ssh access to admin and runtime nodes, defaults to 0.0.0.0/0
-CLOUDWATCH_NAMESPACE=""                  # optional, The namespace for the metrics pushed to CloudWatch
-ENABLE_CLOUDWATCH_LOGS=false             # Required, Send application logs to cloudwatch
-METRICS_SCRAPE_INTERVAL_IN_SECONDS="30"  # Required, How often to scrape data from Curity's metrics endpoint (in seconds)
+AWS_DEPLOYMENT_REGION=""                  # AWS region where the resources would be created, if left empty then the default region configured for the AWS CLI would be used.
```

Comment:
Minor but i would name it AWS_REGION or just REGION

Reply:
I had it initially as `AWS_REGION ` but that seems to cause some conflicts with CDK. I will try with `REGION`